### PR TITLE
add link to instructions for favorited recipes

### PR DESCRIPTION
**Pull request by techprojsforfun**
added link to website with instructions for favorited recipes

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -1,4 +1,11 @@
 class PagesController < ApplicationController
   def home
+    if current_user and current_user.recipes.blank?
```
Comment (omidafshar) [inline app/controllers/pages_controller.rb:3]: This is a great addition! Now the user won't be directed to the log out screen if they had a previous session

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -166,6 +183,7 @@ def update
 		current_user.time = params[:time].to_i
 		current_user.gender = params[:gender]
 		current_user.exercise = params[:exercise]
+		current_user.goal = params[:goal]
```
Comment (omidafshar) [inline app/controllers/users_controller.rb:186]: nice addition, we weren't saving this value before

File: db/migrate/20190512114457_remove_budget_from_users.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class RemoveBudgetFromUsers < ActiveRecord::Migration
+  def change
+    remove_column :users, :budget, :integer
```
Comment (omidafshar) [inline db/migrate/20190512114457_remove_budget_from_users.rb:3]: A legacy attribute that we forgot to remove, good catch

**Review by omidafshar — APPROVED** [review]

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -15,7 +25,7 @@ def show
 		if current_user.recipes.where(:type => "PlanRecipe").blank?
 			@preference_list = [@gender, @weight, @height, @age, @exercise, @goal]
 			@calories = self.class.calc_calories(@preference_list).round(0)
-			@all_recipes = Recipe.find_in_api(@calories, @budget, @time, @dietary_preferences)
+			@all_recipes = Recipe.find_in_api(@calories, @time, @dietary_preferences)
```
Comment (ahz0001) [inline app/controllers/users_controller.rb:28]: Good catch on the budget.

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -1,4 +1,14 @@
 class UsersController < ApplicationController
+
+	before_action :require_login
+
+	def require_login
+		if current_user.nil?
```
Comment (ahz0001) [inline app/controllers/users_controller.rb:6]: Good to see this situation being covered.

**Review by ahz0001 — APPROVED** [review]

**Pull request merged**